docs/source/manifest_validation.rst

```diff
@@ -0,0 +1,211 @@
+.. _Validating a Metadata Manifest:
+
+Validating a Metadata Manifest
+=================================================
+
+Prerequisites
+-------------
+
+**Obtain Synapse Credentials**:
+Ensure you have a Synapse account and set up Synapse configuration file correctly. See the :ref:`installation` section for more details.
+
+**Before Using the Schematic CLI**
+
+- **Install and Configure Schematic**:
+  Ensure you have installed `schematic` and set up its dependencies.
+  See the :ref:`installation` section for more details.
+
+- **Understand Important Concepts**:
+  Familiarize yourself with key concepts outlined on the :ref:`index` of the documentation.
+
+- **Configuration File**:
+  For more details on configuring Schematic, refer to the documentation on :ref:`creating a configuration file for schematic <configuration>`.
+
+- **Obtain a manifest**:
+  Please obtain a manifest by following the documentation of :ref:`generating a manifest <manifest_generation>`.
+
+
+**Using the Schematic API in Production**
+
+Visit the **Schematic API (Production Environment)**:
+`<https://schematic.api.sagebionetworks.org/v1/ui/#/>`_
+
+This will open the **Swagger UI**, where you can explore all available API endpoints.
+
+
+Requirements
+-------------------------------------------------
+
+Authentication
+~~~~~~~~~~~~~~~~~~~~
+Authentication with Synapse is required for metadata validation that includes Cross Manifest Validation rules or the ``filenameExists`` rule.
+
+File Format
+~~~~~~~~~~~~~~
+In general, metadata manifests must be stored as ``.CSV`` files. When validating through the api, manifests may alternatively be sent as a JSON string.
+
+Required Column Headers
+~~~~~~~~~~~~~~~~~~~~~~~~~
+A ``Component`` column that specifies the data type of the metadata must be present in the manifest. Additionally, columns must be present for each attribute in the component that you wish to validate.
+
+Restricted Column Headers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The columns ``id`` and ``entityId`` are reserved for use by schematic and should not be specified in a data model.
+``id`` is used to store a uuid for each row in the manifest.
+``entityId`` is used to store the synId of each entity in the manifest.
+
+Manifest Validation
+-------------------------------------------------
+Overview
+~~~~~~~~~
+Invalidities within a manifest's metadata is classified as either an error or a warning. Depending on the rule itself, whether or not the attribute is required, and what the data modeler has specified.
```

Comment:
I am a bit confused reading this. How about: Invalidities within a manifest’s metadata are classified as either errors or warnings, depending on the rule itself, whether the attribute is required, and what the data modeler has specified.